docs/sources/reference/components/beyla/beyla.ebpf.md

```diff
@@ -156,6 +166,40 @@ Name               | Type           | Description
 `owner_name`       | `string`       | Regular expression of Kubernetes owners of running Pods to match.                                           | `""`    | no
 `pod_labels`       | `map(string)`  | Key-value pairs of labels with keys matching Kubernetes Pods with the provided value as regular expression. |  `{}`   | no
 
+### prometheus block
+
+This block configures which metrics Beyla collects.
+
+Name              | Type           | Description                                                    | Default | Required
+------------------|----------------|----------------------------------------------------------------|---------|---------
+`features`        | `list(string)` | List of features to enable for the Prometheus metrics.         | `["application"]`    | no
+`instrumentations`| `list(string)` | List of instrumentations to enable for the Prometheus metrics. | `["*"]`    | no
+
+`features` is a list of features to enable for the Prometheus metrics. The following features are available:
+
+- `application` exports application-level metrics.
+- `application_span`exports application-level metrics in traces span metrics format.
+- `application_service_graph` exports application-level service graph metrics.
+-  `application_process` exports metrics about the processes that run the instrumented application.
+- `network`, exports network-level metrics.
+
+`instrumentations` is a list of instrumentations to enable for the Prometheus metrics. The following instrumentations are available:
+
+- `*` enables all `instrumentations`. If `*` is present in the list, the other values are simply ignored.
+- `http` enables the collection of HTTP/HTTPS/HTTP2 application metrics.
+- `grpc` enables the collection of gRPC application metrics.
+- `sql` enables the collection of SQL database client call metrics.
+- `redis` enables the collection of Redis client/server database metrics.
+- `kafka` enables the collection of Kafka client/server message queue metrics.
```

Comment:
ideally any field that can only take a specific set of values should be checked in the Validate() function.